### ISSUE TITLE
Likers is null when using GetUserMediaAsync()

### ISSUE THREAD

**Issue by marcelocferdeveloper** [a-legotin/InstaSharper#200]
**Before creating issue**
Put `x` marks below to validate what you've done before posting issue.
 
[x] I've checked [FAQ](https://github.com/a-legotin/InstaSharper/wiki/FAQ)
[x] I've checked [wiki page](https://github.com/a-legotin/InstaSharper/wiki)
[x] I've examined [examples project](https://github.com/a-legotin/InstaSharper/tree/develop/InstaSharper.Examples)
[x] I've did search [existing issues](https://github.com/a-legotin/InstaSharper/issues) for similar problems
[x] I've understand that issue will be closed and locked if issue related to `checkpoint, challenge, feedback required` or `sentry_block` or `signup_block`- see [FAQ](https://github.com/a-legotin/InstaSharper/wiki/FAQ)

**Describe the issue**

When I using GetUserMediaAsync(username) , I get a lot of information about the image, but the property Likes is always null. So, I have to pass the Image ID to GetMediaLikersAsync and get the Likers.
I imagin that is something wrong. Or its not ready yet ?

**Expected behavior**
Get a List of Likers

**Target (please complete the following information):**
 - Platform: Xamarin 
 - Target framework [netstandard]
 - Version [1.5.0]

**Comment by etaxi341** [comment]
With "GetUserMediaAsync(...)" you are only loading the pictures on the profile with the few information you get when scrolling through a profile. So only likecount will be filled but likers not because otherwise the API would have to send another request for more details of the specific image.

**Comment by marcelocferdeveloper** [comment]
@etaxi341 Yes, but for instance has a property containing who have been commented the picture and not only the numbers of comments. 
I checked and saw the property Likers and thought that maybe its in development or not working well.
